core/src/cache.rs

```diff
@@ -1948,16 +2005,18 @@ pub trait ImportResolver {
     /// already transformed in the cache and do not need further processing.
     fn resolve(
         &mut self,
+        pos_table: &mut PosTable,
         import: &term::Import,
         parent: Option<FileId>,
+        //TODO[RFC007]: do we really need a position here, or just a pos idx?
```

Comment:
Just a pos idx, I think. This is only called once, and the pos at the callsite is `&value.pos(pos_table)`